code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/impl/AdminAuthLookUp.java

```diff
@@ -29,35 +29,36 @@
 
     @Override
     public AccountAccess getAccountAccess(ApiRequest request) {
-        AccountAccess accountAccess = null;
         if (SECURITY.get()){
-            return accountAccess;
+            return null;
         }
-
         String authHeader = StringUtils.trim(request.getServletContext().getRequest().getHeader(ENFORCE_AUTH_HEADER));
         if (StringUtils.equals("true", authHeader)) {
-            return accountAccess;
-        }
-        Account admin = authDao.getAdminAccount();
-        if (admin == null){
             return null;
         }
         String projectId = request.getServletContext().getRequest().getHeader(ProjectConstants.PROJECT_HEADER);
-        Account project = null;
         if (projectId == null || projectId.isEmpty()) {
             projectId = request.getServletContext().getRequest().getParameter("projectId");
         }
-        if (projectId != null && !projectId.isEmpty() && !projectId.equalsIgnoreCase(ProjectConstants.USER)) {
+        return getAccountAccessInternal(projectId);
+    }
+
+    private AccountAccess getAccountAccessInternal(String projectId){
+        Account admin = authDao.getAdminAccount();
+        if (admin == null){
+            return null;
+        }
+        Account project = null;
+        if (projectId != null && !projectId.isEmpty()) {
```

Comment:
haven't had time to look at functional change, but it looks like we've taken out the:
!projectId.equalsIgnoreCase(ProjectConstants.USER)
check.  is this intentional?